dartsim/src/GzCollisionDetector.hh

```diff
@@ -15,15 +15,50 @@
  *
 */
 
+#ifndef GZ_PHYSICS_DARTSIM_SRC_GZCOLLISIONDETECTOR_HH_
+#define GZ_PHYSICS_DARTSIM_SRC_GZCOLLISIONDETECTOR_HH_
+
 #include <limits>
 #include <memory>
 
+#include <dart/collision/CollisionResult.hpp>
 #include <dart/collision/ode/OdeCollisionDetector.hpp>
+#include <dart/collision/bullet/BulletCollisionDetector.hpp>
 
 namespace dart {
 namespace collision {
 
-class GzOdeCollisionDetector : public dart::collision::OdeCollisionDetector
+class GzCollisionDetector
+{
+  /// \brief Set the maximum number of contacts between a pair of collision
+  /// objects
+  /// \param[in] _maxContacts Maximum number of contacts between a pair of
+  /// collision objects.
+  public: virtual void SetCollisionPairMaxContacts(std::size_t _maxContacts);
+
+  /// \brief Get the maximum number of contacts between a pair of collision
+  /// objects
+  /// \return Maximum number of contacts between a pair of collision objects.
+  public: virtual std::size_t GetCollisionPairMaxContacts() const;
```

Comment:
include `<cstdio>`

Reply:
added in [fa9468f](https://github.com/gazebosim/gz-physics/pull/593/commits/fa9468f8b95a886bcc856aaf3aa0f94625625e65)